src/main/java/org/spongepowered/common/mixin/api/mcp/world/raid/Raid_StatusMixin_API.java

```diff
@@ -0,0 +1,37 @@
+package org.spongepowered.common.mixin.api.mcp.world.raid;
+
+import org.spongepowered.api.CatalogKey;
+import org.spongepowered.api.data.type.RaidStatus;
+import org.spongepowered.api.plugin.PluginContainer;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+import org.spongepowered.common.SpongeImplHooks;
+import org.spongepowered.common.bridge.CatalogKeyBridge;
+
+@Mixin(targets = "net/minecraft/world/raid/Raid$Status")
+public abstract class Raid_StatusMixin_API implements RaidStatus, CatalogKeyBridge {
+
+    @Inject(at = @At("HEAD"), method = "<init>()V")
```

Comment:
Injections would reside in core mixins, not API mixins, it's to keep separate from implementation versus implementing the api methods themselves.